flit-client/src/components/DialogLine/index.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import face from './face.svg';
 import './styles.css';
+import {FlitUser, FlitMessage} from 'exportedTypes';
+import LineStatus from 'components/LineStatus';
 
 type Props = {
-    // avatar: HTMLElement, // unused for now
-    name: string,
-    line: string,
-    timestamp: Date,
+    user: FlitUser,
+    msg: FlitMessage,
+    msgStatus: string,
+    own: boolean,
 }
 
+const avatarBase:string = `${process.env.PUBLIC_URL}/avatars`
```

Comment:
I really like that we've moved the avatars to exist as assets served by the web server; that makes a ton of sense to me.

Thinking about the structure of the system, we probably want to pass the fully-qualified URLs into this component rather than having this component know anything about where they live. That will make it easier to support avatars that live elsewhere in the future.